packages/block-library/src/cover/deprecated.js

```diff
@@ -152,7 +159,7 @@ const deprecated = [
 			);
 
 			return (
-				<div className={ classes } style={ style }>
+				<div { ...useBlockProps.save( { className: classes, style } ) }>
```

Comment:
I copied this from the `save` function prior to the srcset change. The `useBlockProps` must have been added in addition to the `isRepeated` before the srcset change was merged.